R/pkg/R/DataFrame.R

```diff
@@ -65,6 +65,49 @@ dataFrame <- function(sdf, isCached = FALSE) {
 
 ############################ SparkDataFrame Methods ##############################################
 
+#' Return an SparkDataFrame's name.
+#'
+#' @param x The SparkDataFrame whose name is returned.
+#' @rdname name
+#' @examples
+#'\dontrun{
+#' sparkR.session()
+#' path <- "path/to/file.json"
+#' df <- read.json(path)
+#' df.name()
+#'}
+#' @aliases name,SparkDataFrame-method
+#' @noRd
+#' @note explain since 2.1.0
+setMethod("name",
+          signature(x = "SparkDataFrame"),
+          function(x) {
+            callJMethod(x@sdf, "name")
+          })
+
+#' Set an SparkDataFrame's name.
+#'
+#' @param x The SparkDataFrame whose name is to be set.
+#' @param name The SparkDataFrame name to be set.
+#' @return a new SparkDataFrame renamed.
+#' @rdname setName
```

Comment:
instead of `name` and `setName`, generally it is more R-like to have a setter function - you can see an example at L310 `setMethod("names<-",`

in such a case, it would be better to put both onto the same page by having the same `#' @rdname name` tag